util/delete-entities.py

```diff
@@ -0,0 +1,51 @@
+"""Delete all entities in a workspace.
+
+This is useful when working on Export to Saturn feature.
+
+To run:
+- virtualenv ~/virtualenv/firecloud
+- source ~/virtualenv/firecloud/bin/activate
+- pip install firecloud
+- python util/delete-entities.py WORKSPACE_NAMESPACE WORKSPACE_NAME
+"""
+
+import sys
+
+from firecloud import fiss
+
+
+def main():
+    if len(sys.argv) != 3:
+        raise ValueError(
+            'Usage: python util/delete-entities.py WORKSPACE_NAMESPACE WORKSPACE_NAME'
+        )
+
+    workspace_namespace = sys.argv[1]
+    workspace_name = sys.argv[2]
+
+    args = type('obj', (object, ), {
+        'project': workspace_namespace,
+        'workspace': workspace_name
+    })
+    entities = fiss.entity_list(args)
+
+    # entities are sorted by type: participant, participant_set, sample,
+    # sample_set. FireCloud complains if we delete a participant before deleting
+    # associated participant_set/sample/sample_set. By reversing the list, we
+    # won't have this problem. We will delete: sample_set, sample,
+    # participant_set, participant.
+    for entity in reversed(entities):
+        print('Deleting ' + entity)
+        args = type(
+            'obj', (object, ), {
```

Comment:
I would probably just extract the namedtuple out to the top level:

```
Args = collections.namedtuple('Args', ['project', 'workspace', 'yes', 'entity_type', 'entity'])
```

Then above:
```
entities = fiss.entity_list(Args(workspace_namespace, workspace_name))
```

and here:
```
entity_split entity.split('\t')
fiss.entity_delete(Args(workspace_namespace, workspace_name, True, entity_split[0], entity_split[1]))
```

Could also use keyword arguments if you think that makes it clearer.

Reply:
Done